docs/static/releasenotes.asciidoc

```diff
@@ -13,6 +14,52 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+
+[[logstash-6-3-0]]
+=== Logstash 6.3.0 Release Notes
+
+[NOTE]
+--
+Persistent Queue users must upgrade. Old data will not be compatible with 6.3.0, and must be migrated or deleted. Read
+{logstash-ref}}/upgrading-logstash-pqs.html[Upgrading Persistent Queue from Logstash 6.2.x and Earlier]
```

Comment:
- Delete extra `}` after attribute in link format `{logstash-ref}}`
- Consider changing NOTE to IMPORTANT 

- Question: We merged the PQ upgrade content into 6.2 and 6.3 only.  At 6.x and beyond, the link ({logstash-ref}/upgrading-logstash-pqs.html) will direct people back to the old content. Is this a case for hard-coding version number(s) in the link?  We've published that link in a lot of places--emails, blogs, and now release notes.   
I would appreciate input on the best approach for managing this in a way that makes sense across versions. @dedemorton @andrewvc